code/script.js

```diff
@@ -15,37 +20,190 @@ const showMessage = (message, sender) => {
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/usercoffee.png" alt="User" />  
```

Comment:
I LOVE that you used your own little images for the chat bubbles, instead of the already existing ones! They're really cute too!